api/shared/nodenetworkconfigurationpolicy_types.go

```diff
@@ -12,18 +12,21 @@ type NodeNetworkConfigurationPolicySpec struct {
 	// The desired configuration of the policy
 	DesiredState State `json:"desiredState,omitempty"`
 
-	// When set to true, changes are applied to all nodes in parallel
+	// Percentage of matching cluster nodes that may apply policy configuration concurrently.
+	// If not specified, policy is applied one node at a time.
+	// Value of 0 means all nodes apply policy in parallel,
+	// Value of n, n > 1 means that up to n% of cluster nodes may apply policy configuration concurrently.
 	// +optional
-	Parallel bool `json:"parallel,omitempty"`
+	ChunkSize *int `json:"chunkSize,omitempty"`
```

Comment:
I am super worry about upgrades here I think we have release a versin with the `Parallel` parameter, we have to read operator-sdk doc but I think we have to add ChickSize and mark Parallel as obsolete and at the next release delete Parallel, also maybe even add migration code that in case of Parallel argument we conert it into ChucnkSize also we may need to move this tov1btea2 or something like this, we will need to test this with the upgrae test, but let's sync first.

I have an idea maybe both make sense, If Parallel is false is pure sequencial rollouthttps://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/4617/pull-kubevirt-e2e-kind-1.17-sriov/1336545341402517504 and if Paralle is true it means we run the NNCP inparallel and in that case ChunkSize can be configured.